### PR TITLE
configuration: add nemu support

### PR DESCRIPTION
**Pull request by egernst**
NEMU requires a unique runtime configuration.  Add support for utilizing
a configuration-nemu.toml

Fixes: #1647

Signed-off-by: Eric Ernst <eric.ernst@intel.com>

### CODE REVIEW

**Comment by egernst** [comment]
None of the functional tests will exercise this today.  Going forward, our NEMU job should make use of the resulting nemu toml.

/cc @jcvenegas @chavafg @GabyCT @mcastelino

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .git-commit
 .git-commit.tmp
 /cli/config/configuration-fc.toml
+/cli/config/configuration-nemu.toml
```
Comment (jcvenegas) [inline .gitignore]: You will need to modify 
https://github.com/kata-containers/runtime/blob/master/cli/config/configuration-qemu.toml.in#L15

File: cli/config/configuration-nemu.toml.in [hunk 2]
```diff
@@ -0,0 +1,331 @@
+# Copyright (c) 2017-2019 Intel Corporation
```
Comment (WeiZhang555) [inline cli/config/configuration-nemu.toml.in]: Since 2017~2018 are passed, I think this should be 2019?

Comment (grahamwhaley) [reply cli/config/configuration-nemu.toml.in]: If we are inheriting things from files from the past, it is normal to have a year range covering all the years bits came from

**Pull request merged**